docs/install_guides/installing-model-dev-tools.md

```diff
@@ -47,16 +54,16 @@ To create a virtual environment, use the following command:
 .. tab:: Linux and macOS
 
    .. code-block:: sh
-   
+
       source openvino_env/bin/activate
-   
+
 .. tab:: Windows
 
    .. code-block:: sh
-   
+
       openvino_env\Scripts\activate
-     
-     
+
+
 @endsphinxdirective
 
 
```

Comment:
we should also copy Step 4 and 5 from the PR for master.

Reply:
Done in 
https://github.com/openvinotoolkit/openvino/pull/13868/commits/e98da6c9272054071ed559846969f2697b0a358b
https://github.com/openvinotoolkit/openvino/pull/13868/commits/c0c50a5be2d92d9bceaed5f449f6d470800d768d
https://github.com/openvinotoolkit/openvino/pull/13868/commits/f79377ba7a72df532950014a58e1a41f4a76b621